src/components/__tests__/Slider.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { shallow, ShallowWrapper } from "enzyme";
+import { createElement, DOM } from "react";
+
+import * as RcSlider from "rc-slider";
+
+import { Slider as SliderComponent, SliderProps } from "../Slider";
+
+describe("Slider", () => {
+    let sliderProps: SliderProps;
+    let slider: ShallowWrapper<SliderProps, any>;
+    const value = 20;
+    const maxValue = 100;
+    const minValue = 0;
+    const stepValue = 1;
+    const noOfMarkers = 0;
+    beforeEach(() => {
+        sliderProps = {
+            disabled: false,
+            maxValue,
+            minValue,
+            noOfMarkers,
+            stepValue,
+            tooltipText: "{1}",
+            value
+        };
+    });
+    const renderSlider = (props: SliderProps) => shallow(createElement(SliderComponent, props));
+
+    it("renders the structure", () => {
+        slider = renderSlider(sliderProps);
+
+        expect(slider).toBeElement(
+            DOM.div({ className: "widget-slider" },
+                createElement(RcSlider, {
+                    disabled: false,
+                    included: true,
+                    max: maxValue,
+                    min: minValue,
+                    step: stepValue,
+                    tipFormatter: jasmine.any(Function) as any,
+                    value,
+                    vertical: false
+                })
+            )
+        );
+    });
+
+    describe("with a value", () => {
+        it("renders a slider with the given value", () => {
+            slider = renderSlider(sliderProps);
+            const RcSliderComponent = slider.find(RcSlider);
+
+            expect(RcSliderComponent.props().value).toBe(sliderProps.value);
+        });
+
+        it("that is undefined renders a slider with the calculated default value", () => {
+            sliderProps.value = undefined;
```

Comment:
The `describe` block tells me that this is a test with a value and here you pass `undefined`. The name of the test is also an indication that the test doesn't fit well.